pubtools/pulplib/_impl/model/repository/file.py

```diff
@@ -20,3 +30,77 @@ class FileRepository(Repository):
     )
 
     mutable_urls = attr.ib(default=attr.Factory(lambda: ["PULP_MANIFEST"]), type=list)
+
+    def upload_file(self, file_obj, relative_url=None):
+        """Upload a file to this repository.
+
+        Args:
+            file_obj (str, file object)
+                If it's a string, then it's the path of file to upload.
+                Else, it ought to be a file-like object, see:
+                https://docs.python.org/3/glossary.html#term-file-object
```

Comment:
Could you please use a hyperlink instead of just adding the URL in the doc string?

What I mean is, the user should see "file-like object" and if they click on that, they should be directed to that doc.

Reply:
yep.